src/net_processing.cpp

```diff
@@ -743,11 +743,11 @@ void UpdateLastBlockAnnounceTime(NodeId node, int64_t time_in_seconds)
     if (state) state->m_last_block_announcement = time_in_seconds;
 }
 
-// Returns true for outbound peers, excluding manual connections, feelers, and
-// one-shots
+// Returns true for outbound transaction-relay peers, excluding manual
```

Comment:
nit: comment is a bit misleading and without context lets suggest we have already transaction-only peers, isn't full-relay peers a better wording ?

Reply:
Addressed in latest commit.